src/ApplicationSampler.cpp

```diff
@@ -281,19 +283,23 @@ namespace geopm
 
     double ApplicationSamplerImp::cpu_hint_time(int cpu_idx, uint64_t hint) const
     {
+        double result = NAN;
```

Comment:
Shouldn't this just be 0?  I think that would fix the problem with the totals.

Reply:
No, the NaN is used to mark the signal to be ignored by the aggregation function.